hw/ip/flash_ctrl/data/flash_ctrl_testplan.hjson

```diff
@@ -419,5 +419,30 @@
             - update_err: A shadow register encountered an update error.
             '''
     }
+    {
+      name: eviction_cg
+      desc: '''
+            Covers eviction with mp_region_cfgs for data and info regions.
+            Sample all 4 rd_buf status.
+            When each buffer hazard is set, capture the address stored in the buffer.
+            Then search from tb data base to see which region the address belong to.
+            After that record the config value (scrambe_en and ecc_en) of the region.
+            Use cross over buffer index, the operation to cause the eviction and
+            the region config values.
+            '''
+    }
+    {
+      name: fetch_code_cg
+      desc: '''
+            Covers whether dut is received valid / invalue key value from ral.exec
+            register. Cross with tlul.instr_types.
+            '''
+    }
+    {
+      name: rma_init_cg
+      desc: '''
+            Cover rma operation is executed before or after flash_init started.
```

Comment:
Could you clarify this description? The covergroup seems to capture state, is it before or after, or during?

Reply:
RMA can be performed regardless of flash init.
So purpose of this cover group is to show rma can be performed either
'before' or 'during' of rma.
After RMA is not possible because flash become disable state.
So that case is excluded from sampling.